php/comet/start.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Comet\Comet;
+
+$app = new Comet([ 
+	'port' => 3000,
+	shell_exec('nproc') ?? 32,
```

Comment:
rather use environment variable (like for `swoole` based **frameworks**, simps, hyperf ...)